.github/workflows/test-env-deploy.yml

```diff
@@ -57,11 +57,21 @@ jobs:
           pattern: (http|https)://[a-zA-Z0-9.-]+/graphql/?
         run: |
           echo "::set-output name=custom_api_uri::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
+      - name: Get MARKETPLACE_URL
+        id: marketplace_url
+        # Search for MARKETPLACE_URL in PR description
+        env:
+          pull_request_body: ${{ github.event.pull_request.body }}
+          prefix: MARKETPLACE_URL=
+          pattern: (http|https)://[a-zA-Z0-9.-]+/?
+        run: |
+          echo "::set-output name=custom_marketplace_url::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
 
       - name: Run build
         env:
           # Use custom API_URI or the default one
           API_URI: ${{ steps.api_uri.outputs.custom_api_uri || 'https://qa.staging.saleor.cloud/graphql/' }}
+          MARKETPLACE_URL: ${{ steps.marketplace_url.outputs.custom_marketplace_url }}
```

Comment:
Please remember to set MARKETPLACE_URL for staging ad cloud deployment workflows